### PR TITLE
recipes: new recipe `coincurve`

### PR DESCRIPTION
**Pull request by T-Dynamos**
Hello!

### CODE REVIEW

File: pythonforandroid/recipes/coincurve/__init__.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import os
+from pythonforandroid.recipe import PythonRecipe
+
+
+class CoincurveRecipe(PythonRecipe):
+    version = '18.0.0'
+    url = 'https://github.com/ofek/coincurve/archive/v{version}.tar.gz'
+    call_hostpython_via_targetpython = False
+    depends = [('python2', 'python3crystax'), 'setuptools',
```
Comment (misl6) [inline pythonforandroid/recipes/coincurve/__init__.py:9]: Hi @T-Dynamos !

`python2` and `python3crystax` are not in our recipe list anymore.

Can you please update the recipe accordingly?

**Review by misl6 — CHANGES_REQUESTED** [review]

**Comment by AndreMiras** [comment]
Hi @T-Dynamos, thanks for the PR, sorry for the delay, the PR got buried under.
Is the recipe still relevant?
If so would you mind rebasing so that it triggers a build.
```
git pull --rebase upstream develop # assuming upstream is the remote name
```

**Comment by T-Dynamos** [comment]
@AndreMiras Done!.

**Comment by AndreMiras** [comment]
Awesome job, thanks!

**Pull request merged**